app/routes/login.tsx

```diff
@@ -123,20 +200,25 @@ export default function Login() {
 									},
 								})}
 							/>
-							{!userCreated && (
+							{(userCreated === 0 || signupAllowed) && (
 								<span
 									className={`
-								absolute right-2 top-1/2 -translate-y-1/2
-								w-2 h-2 rounded-full
-								${$password >= 8 ? "bg-green-600" : "dark:bg-neutral-700 bg-neutral-400"}
-							`}
+                    absolute right-2 top-1/2 -translate-y-1/2
+                    w-2 h-2 rounded-full
+                    ${$password >= 8 ? "bg-green-600" : "dark:bg-neutral-700 bg-neutral-400"}
+                  `}
 									aria-hidden="true"
 								/>
 							)}
 						</div>
 
 						<Button type="submit">
```

Comment:
```suggestion
						<Button className="gap-1" type="submit">
```
Instead of the `ml-1` you're using